Tests/FluentAssertions.Specs/Execution/CallerIdentifierSpecs.cs

```diff
@@ -259,8 +259,11 @@ public void When_arguments_contain_verbatim_string_it_should_include_that_to_the
             var foo = new Foo();
 
             // Act
+            // ReSharper disable RedundantStringInterpolation
```

Comment:
Let's add these exception to the top of the file with the other analyzer exclusions.
https://github.com/fluentassertions/fluentassertions/blob/e27ee6e75b1bee897abc4b880224a154b93bc623/Tests/FluentAssertions.Specs/Execution/CallerIdentifierSpecs.cs#L13

Reply:
Fixed in [899d468](https://github.com/fluentassertions/fluentassertions/pull/2470/commits/899d46880132f777a1a979dd4fa23a357b5a30ae).